<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>430</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Number</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Notes</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="addContactButton">
        <property name="text">
         <string>Add Contact</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteContactButton">
        <property name="text">
         <string>Delete Contact</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editContactButton">
        <property name="text">
         <string>Edit Contact</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchContactButton">
        <property name="text">
         <string>Search Contact</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addNoteButton">
        <property name="text">
         <string>Add Note</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteNoteButton">
        <property name="text">
         <string>Delete Note</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadFromFileButton">
        <property name="text">
         <string>Load from File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveToFileButton">
        <property name="text">
         <string>Save to File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
